api/routers/areas.py

```diff
@@ -87,21 +87,40 @@ async def create_area(new_area: AreaConfigDTO, reboot_processor: Optional[bool]
             detail=bad_request_serializer("Area with ID: 'ALL' is not valid.", error_type="Invalid ID")
         )
 
-    cameras = [x for x in config_dict.keys() if x.startswith("Source_")]
-    cameras = [map_camera(x, config_dict, []) for x in cameras]
-    camera_ids = [camera["id"] for camera in cameras]
+    cameras = config.get_video_sources()
+    camera_ids = [camera.id for camera in cameras]
     if not all(x in camera_ids for x in new_area.cameras.split(",")):
         non_existent_cameras = set(new_area.cameras.split(",")) - set(camera_ids)
         raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail=f"The cameras: {non_existent_cameras} do not exist")
+    occupancy_rules = new_area.occupancy_rules
+    del new_area.occupancy_rules
     area_dict = map_to_config_file_format(new_area)
+
+    config_dict = extract_config()
     config_dict[f"Area_{len(areas)}"] = area_dict
     success = update_config(config_dict, reboot_processor)
+
+    if occupancy_rules:
+        area_config_path = get_config().get_area_config_path(new_area.id)
+        Path(os.path.dirname(area_config_path)).mkdir(parents=True, exist_ok=True)
+
+        if os.path.exists(area_config_path):
+            with open(area_config_path, "r") as area_file:
+                data = json.load(area_file)
+        else:
+            data = {}
+
+        with open(area_config_path, "w") as area_file:
+            data["occupancy_rules"] = occupancy_rules.to_store_json()["occupancy_rules"]
+            json.dump(data, area_file)
+
     if not success:
         return handle_response(area_dict, success, status.HTTP_201_CREATED)
 
     area_directory = os.path.join(os.getenv("AreaLogDirectory"), new_area.id, "occupancy_log")
     Path(area_directory).mkdir(parents=True, exist_ok=True)
 
+    # known issue: Occupancy rules not returned
     return next((area for area in get_areas() if area["id"] == area_dict["Id"]), None)
```

Comment:
Maybe we can create a ticket to fix it.
I think that it can be fixed easily adding  in the `get_areas` function a parameter called `include_occupancy_rules=False`

Reply:
It would be fixed if we migrate from the .ini to a .json for all areas, right? That's what we discussed on the daily

Reply:
This `get_areas` function is local in this file, not the one from config_engine. I don't think we should add that logic here as well.

I can create a ticket for it